h2o-core/src/main/java/water/AutoBuffer.java

```diff
@@ -972,13 +976,14 @@ int read( byte[] buf, int off, int len ) {
   // Utility functions to handle common UDP packet tasks.
   // Get the 1st control byte
   int  getCtrl( ) { return getSz(1).get(0)&0xFF; }
+  // Get unique ID in next 2 bytes
+  char getUniqueId() { return getSz(1+2).getChar(1);}
```

Comment:
`getUniqueId` is on purpose where `getPort` was previously. In code, `getPort` is used to skip bytes before the entry and the bytes for port itself. This was `getPort` will still have the same side-effect except we skip also the nodeUniqueId info which is desired

Reply:
could you please include the comment in the codebase?

Reply:
done